File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/GroupBox/Features/BackgroundPage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;

using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.GroupBox.Features
{
	public partial class BackgroundPage : UserControl
	{
		public BackgroundPage()
		{
			InitializeComponent();
		}

        /// <summary>
        /// Handles CheckedChanged event of the RadioButton controls
        /// Changes GroupBox background color according to RadioButton
        /// value that raised the event
        /// </summary>
		private void Color_Changed(object sender, EventArgs e)
		{
            if (sender == mobjMoccasin)
				mobjGroupColor.BackColor = Color.Moccasin;
			else if(sender == mobjSnow)
				mobjGroupColor.BackColor = Color.Snow;
			else if(sender == mobjTransparent)
				mobjGroupColor.BackColor = Color.Transparent;
		}
	}
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/GroupBox/Features/BackgroundPage.vb
Imports System.Drawing

Namespace CompanionKit.Controls.GroupBox.Features

    Public Class BackgroundPage

        Public Sub New()

            ' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()

        End Sub
        ''' <summary>
        ''' Handles CheckedChanged event of the RadioButton controls
        ''' Changes GroupBox background color according to RadioButton
        ''' value that raised the event
        ''' </summary>
        Private Sub Color_Changed(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles mobjTransparent.CheckedChanged, mobjSnow.CheckedChanged, mobjMoccasin.CheckedChanged
            If (sender Is Me.mobjMoccasin) Then
                Me.mobjGroupColor.BackColor = Color.Moccasin
            ElseIf (sender Is Me.mobjSnow) Then
                Me.mobjGroupColor.BackColor = Color.Snow
            ElseIf (sender Is Me.mobjTransparent) Then
                Me.mobjGroupColor.BackColor = Color.Transparent
            End If
        End Sub
    End Class

End Namespace